src/common/ceph_strings.cc

```diff
@@ -77,9 +77,17 @@ const char *ceph_osd_alloc_hint_flag_name(int f)
 		return "shortlived";
 	case CEPH_OSD_ALLOC_HINT_FLAG_LONGLIVED:
 		return "longlived";
-	case CEPH_OSD_ALLOC_HINT_FLAG_COMPRESSIBLE:
```

Comment:
Hmm, do we really want to remove the alloc_hint flags, or can they coexist?  The nice thing about the existing ones is that they are persistent, so we have the [in]compressible information at any time (not just for a given write).